src/test/gcp/cloudscheduler.spec.ts

```diff
@@ -0,0 +1,104 @@
+import * as _ from "lodash";
+import { expect } from "chai";
+import * as nock from "nock";
+import * as api from "../../api";
+
+import { cloudscheduler } from "../../gcp";
+
+const VERSION = "v1beta1";
+
+const testJob = {
+  name: "projects/test-project/locations/us-east1/jobs/test",
+  schedule: "every 5 minutes",
+  timeZone: "America/Los_Angeles",
+  httpTarget: {
+    uri: "https://afakeone.come",
+    httpMethod: "POST",
+  },
+  retryConfig: {},
+};
+
+describe("cloudscheduler", () => {
+  describe("createOrUpdateJob", () => {
+    afterEach(() => {
+      nock.cleanAll();
+    });
+
+    it("should create a job if none exists", async () => {
+      nock(api.cloudschedulerOrigin)
+        .get(`/${VERSION}/${testJob.name}`)
+        .reply(404, { context: { response: { statusCode: 404 } } });
+      const mockJobResp = { schedule: "every 5 minutes" };
+      nock(api.cloudschedulerOrigin)
+        .post(`/${VERSION}/projects/test-project/locations/us-east1/jobs`)
+        .reply(200, mockJobResp);
+
+      const response = await cloudscheduler.createOrReplaceJob(testJob);
+
+      expect(response.body).to.deep.equal(mockJobResp);
+      expect(nock.isDone());
```

Comment:
[`nock.isDone()` returns a boolean](https://github.com/nock/nock#isdone)

`expect(false)` does nothing. An expectation needs to be defined (e.g. `expect(false).is.true`) for it to cause the test to fail.

These apply throughout.